cpp/src/gandiva/lru_cache.h

```diff
@@ -113,9 +196,32 @@ class LruCache {
     lru_list_.erase(i);
   }
 
+  void evictObject() {
+    // evict item from the end of most recently used list
+    typename list_type::iterator i = --lru_list_.end();
+    const size_t size_to_decrease = size_map_.find(*i)->second.first;
+    const value_type value = map_.find(*i)->second.first;
+    cache_size_ = cache_size_ - size_to_decrease;
+    map_.erase(*i);
+    size_map_.erase(*i);
+    lru_list_.erase(i);
+  }
+
+  void evitObjectSafely(size_t object_cache_size) {
```

Comment:
Ditto

Reply:
I checked, and I might be the case, so I capitalized the first letter.